[stdlog] 2022-10-28 21:02:39,710 avocado.test INFO | INIT 1-tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py:Test.test
[stdlog] 2022-10-28 21:02:39,711 avocado.test DEBUG| PARAMS (key=timeout, path=*, default=None) => None
[stdlog] 2022-10-28 21:02:39,711 avocado.test DEBUG| Test metadata:
[stdlog] 2022-10-28 21:02:39,711 avocado.test DEBUG|   filename: /root/GitRepo/anolis-pkg-tests/tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py
[stdlog] 2022-10-28 21:02:39,711 avocado.test DEBUG|   teststmpdir: /var/tmp/avocado_cmoau920
[stdlog] 2022-10-28 21:02:39,711 avocado.test INFO | START 1-tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py:Test.test
[stdlog] 2022-10-28 21:02:39,711 avocado.test INFO | setup
[stdlog] 2022-10-28 21:02:39,711 avocado.test DEBUG| PARAMS (key=remote, path=*, default=None) => None
[stdlog] 2022-10-28 21:02:39,712 avocado.utils.process INFO | Running 'rpm -q util-linux'
[stdlog] 2022-10-28 21:02:39,719 avocado.utils.process DEBUG| [stdout] util-linux-2.32.1-35.0.1.an8.x86_64
[stdlog] 2022-10-28 21:02:39,720 avocado.utils.process INFO | Command 'rpm -q util-linux' finished with 0 after 0.007394428s
[stdlog] 2022-10-28 21:02:39,720 avocado.utils.process INFO | Running 'yum update util-linux -y'
[stdlog] 2022-10-28 21:02:40,003 avocado.utils.process DEBUG| [stdout] Last metadata expiration check: 2:45:08 ago on Fri 28 Oct 2022 06:17:32 PM CST.
[stdlog] 2022-10-28 21:02:40,269 avocado.utils.process DEBUG| [stdout] Dependencies resolved.
[stdlog] 2022-10-28 21:02:40,270 avocado.utils.process DEBUG| [stdout] Nothing to do.
[stdlog] 2022-10-28 21:02:40,270 avocado.utils.process DEBUG| [stdout] Complete!
[stdlog] 2022-10-28 21:02:40,559 avocado.utils.process INFO | Command 'yum update util-linux -y' finished with 0 after 0.838467925s
[stdlog] 2022-10-28 21:02:40,560 avocado.utils.process INFO | Running 'sudo mkswap {{path/to/file}}'
[stdlog] 2022-10-28 21:02:40,571 avocado.utils.process DEBUG| [stderr] mkswap: 
[stdlog] 2022-10-28 21:02:40,571 avocado.utils.process DEBUG| [stderr] cannot open {{path/to/file}}
[stdlog] 2022-10-28 21:02:40,571 avocado.utils.process DEBUG| [stderr] : 
[stdlog] 2022-10-28 21:02:40,571 avocado.utils.process DEBUG| [stderr] No such file or directory
[stdlog] 2022-10-28 21:02:40,572 avocado.utils.process INFO | Command 'sudo mkswap {{path/to/file}}' finished with 1 after 0.011513163s
[stdlog] 2022-10-28 21:02:40,572 avocado.test ERROR| 
[stdlog] 2022-10-28 21:02:40,572 avocado.test ERROR| Reproduced traceback from: /usr/local/lib/python3.6/site-packages/avocado/core/test.py:761
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR| Traceback (most recent call last):
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|   File "/root/GitRepo/anolis-pkg-tests/tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py", line 27, in test
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|     ret_c, ret_o = self.cmd("sudo mkswap {{path/to/file}}")
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|   File "/root/GitRepo/anolis-pkg-tests/tests/common/basetest.py", line 54, in cmd
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|     return self.local.cmd(command, ignore_status=ignore_status)
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|   File "/root/GitRepo/anolis-pkg-tests/tests/common/hosts.py", line 45, in cmd
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|     result = process.run(command, ignore_status=ignore_status, shell=True)
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|   File "/usr/local/lib/python3.6/site-packages/avocado/utils/process.py", line 1015, in run
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR|     raise CmdError(cmd, sp.result)
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR| avocado.utils.process.CmdError: Command 'sudo mkswap {{path/to/file}}' failed.
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR| stdout: b''
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR| stderr: b'mkswap: cannot open {{path/to/file}}: No such file or directory\n'
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR| additional_info: None
[stdlog] 2022-10-28 21:02:40,573 avocado.test ERROR| 
[stdlog] 2022-10-28 21:02:40,573 avocado.test DEBUG| Local variables:
[stdlog] 2022-10-28 21:02:40,585 avocado.test DEBUG|  -> self <class 'tc_ag_util-linux_mkswap_fun_002.Test'>: 1-tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py:Test.test
[stdlog] 2022-10-28 21:02:40,585 avocado.test INFO | teardown
[stdlog] 2022-10-28 21:02:40,585 avocado.utils.process INFO | Running 'yum erase  -y'
[stdlog] 2022-10-28 21:02:40,863 avocado.utils.process DEBUG| [stderr] No packages marked for removal.
[stdlog] 2022-10-28 21:02:40,870 avocado.utils.process DEBUG| [stdout] Dependencies resolved.
[stdlog] 2022-10-28 21:02:40,871 avocado.utils.process DEBUG| [stdout] Nothing to do.
[stdlog] 2022-10-28 21:02:40,871 avocado.utils.process DEBUG| [stdout] Complete!
[stdlog] 2022-10-28 21:02:41,177 avocado.utils.process INFO | Command 'yum erase  -y' finished with 0 after 0.590808822s
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR| Traceback (most recent call last):
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR|   File "/usr/local/lib/python3.6/site-packages/avocado/core/test.py", line 796, in _run_avocado
[stdlog]     raise test_exception
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR|   File "/usr/local/lib/python3.6/site-packages/avocado/core/test.py", line 756, in _run_avocado
[stdlog]     testMethod()
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR|   File "/root/GitRepo/anolis-pkg-tests/tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py", line 27, in test
[stdlog]     ret_c, ret_o = self.cmd("sudo mkswap {{path/to/file}}")
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR|   File "/root/GitRepo/anolis-pkg-tests/tests/common/basetest.py", line 54, in cmd
[stdlog]     return self.local.cmd(command, ignore_status=ignore_status)
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR|   File "/root/GitRepo/anolis-pkg-tests/tests/common/hosts.py", line 45, in cmd
[stdlog]     result = process.run(command, ignore_status=ignore_status, shell=True)
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR|   File "/usr/local/lib/python3.6/site-packages/avocado/utils/process.py", line 1015, in run
[stdlog]     raise CmdError(cmd, sp.result)
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR| avocado.utils.process.CmdError: Command 'sudo mkswap {{path/to/file}}' failed.
[stdlog] stdout: b''
[stdlog] stderr: b'mkswap: cannot open {{path/to/file}}: No such file or directory\n'
[stdlog] additional_info: None
[stdlog] 2022-10-28 21:02:41,178 avocado.test ERROR| ERROR 1-tests/system_base/util-linux/tc_ag_util-linux_mkswap_fun_002.py:Test.test -> CmdError: Command 'sudo mkswap {{path/to/file}}' failed.
[stdlog] stdout: b''
[stdlog] stderr: b'mkswap: cannot open {{path/to/file}}: No such file or directory\n'
[stdlog] additional_info: None
[stdlog] 2022-10-28 21:02:41,178 avocado.test INFO | 
